core_lib/src/tool/handtool.cpp

```diff
@@ -108,17 +108,18 @@ void HandTool::transformView(Qt::KeyboardModifiers keyMod, Qt::MouseButtons butt
         QVector2D startV(getLastPixel() - centralPixel);
         QVector2D curV(getCurrentPixel() - centralPixel);
 
-        float angleOffset = (atan2(curV.y(), curV.x()) - atan2(startV.y(), startV.x())) * 180.0 / M_PI;
+        qreal deltaPoint = static_cast<qreal>(atan2(curV.y(), curV.x()) - atan2(startV.y(), startV.x()));
+        qreal angleOffset = MathUtils::radToDeg(deltaPoint);
```

Comment:
I don't like the naming here as `deltaPoint` isn't a point at all. More accurately it could be called `deltaAngle`, but that's almost the same thing as `angleOffset`. I think we should just use one variable for this.

Reply:
Indeed that makes no sense, and why two variables 🤔